uiowa/uiowa
# SPPA - Private Page Content Type

Closes #3922 



# How to test 
1. `blt ds --site=sppa.uiowa.edu`
2. `yarn workspace sppa_core build`
3. `drush @sppa.local cr`
4. `drush @sppa.local uli`
5. Go to add a private page - should be same experience as adding a page
    1. Same options/blocks/styling/etc
6. Save the page with some content added from the ‘Edit’ section in addition to the ‘Layout’ section 
    1. be sure to add a statistic because had to fix styling to display similar to page
7. Save page
8. You should be able to view the page as a logged in user - if you view the node in incognito or logout - you should view an access denied page that says you’re not authorized to access the page
